Sources/TransferMethod/ListTransferMethodPresenter.swift

```diff
@@ -153,16 +167,21 @@ final class ListTransferMethodPresenter {
         if let transferMethod = getTransferMethod(at: transferMethodIndex),
             let country = transferMethod.getField(fieldName: .transferMethodCountry) as? String,
             let transferMethodType = transferMethod.getField(fieldName: .type) as? String {
-            var lastFourDigitAccountNumber: String?
-            if let lastFourDigit = getLastDigits(transferMethod, number: 4) {
-                lastFourDigitAccountNumber = String(format: "%@%@",
-                                                    "transfer_method_list_item_description".localized(),
-                                                    lastFourDigit)
+            var additionalInfo: String?
+            if transferMethodType == "PAYPAL_ACCOUNT" {
+                additionalInfo = user?.email
+            } else {
+                if let lastFourDigit = getLastDigits(transferMethod, number: 4) {
+                    additionalInfo = String(format: "%@%@",
+                                            "transfer_method_list_item_description".localized(),
+                                            lastFourDigit)
+                }
             }
             return ListTransferMethodCellConfiguration(
                 transferMethodType: transferMethodType.lowercased().localized(),
                 transferMethodCountry: country.localized(),
-                lastFourDigitAccountNumber: lastFourDigitAccountNumber,
+
+                additionalInfo: additionalInfo,
```

Comment:
I am ok with additionalInfo instead of subTitleText

Reply:
ok